autoscheduler/frontend/src/tests/ui/App.test.tsx

```diff
@@ -18,6 +18,10 @@ test('renders without errors', async () => {
   fetchMock.mockResponseOnce(JSON.stringify({}));
   // Mock response for SelectTerm component so that App renders correctly
   fetchMock.mockResponseOnce(JSON.stringify({}));
+  fetchMock.mockResponseOnce(JSON.stringify({})); // Mock api/terms, again
+
+  //  WHY DOES THIS BREAK THINGS
+  // fetchMock.mockIf('api/terms', async () => JSON.stringify({}));
```

Comment:
I tried making `mockIf` and `doMockIf` work for a while and whenever I used either it would crash the test suite. It would be really nice if we could get this to work, but it seems out of our control